config/context/application.properties

```diff
@@ -4,12 +4,13 @@ credential.privkeyFilePath=config/context/priv.key
 brvs.userKeysPath=config/context
 brvs.userKeysCount=10
 brvs.localhostname=localhost
-brvs.userDomains=users
+brvs.userDomains=d3,sora
 quorum.key=user_quorum
-accounts.holder=notary@notary
+accounts.holder=client_accounts@notary
 iroha.host=brvs-iroha
 iroha.port=50051
 mongo.host=brvs-mongodb
 mongo.port=27017
-rmq.host=brvs-rmq
+rmq.host=d3-rmq
 rmq.port=5672
+whitelist.validation=86400
```

Comment:
commit only this change please :)

Reply:
I suggest to keep 'client_accounts@notary' for accounts.holder..

Reply:
Okay 😆